RecoDataProducts/inc/TriggerInfo.hh

```diff
@@ -15,24 +16,27 @@ namespace mu2e {
   class CosmicTrackSeed;
   struct TriggerInfo
   {
-     
+    constexpr static size_t MaxNObj = 20;
+
     const TriggerFlag&             triggerBits()     const { return _triggerBits; }
```

Comment:
Is there a reason to hardcode this to 20?

Reply:
I preferred to use the std::array instead of a std::vector for this data product because I thought that in this developing stage it is important for us to spot when there is a large number of reco-objects trigger matched. 20 seems to be large enough.